code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -295,6 +336,36 @@
 		busy = SPIDER_IDLE
 		stop_automated_movement = FALSE
 
+/datum/action/spider/comm
+	name = "Command"
+	button_icon_state = "cult_comms"
+
+	/datum/action/comm/IsAvailable()
+	if(!mob/living/simple_animal/hostile/poison/giant_spider/nurse/midwife(owner))
+		return FALSE
+	return ..()
+
+	/datum/action/innate/spider/comm/Activate()
+	var/input = stripped_input(usr, "Input a message for your legions to follow.", "Command", "")
+	if(!input || !IsAvailable())
+		return
+
+	spider_command(usr, input)
+
+/proc/spider_command(mob/living/user, message)
+	var/my_message
+	if(!message)
+		return
+	my_message = "<b>COMMAND FROM SPIDER QUEEN:</b> [message]

```

Comment:
missing "

Reply:
i wouldn't review this, i just fixed a huge part of it don't worry

Reply:
i, as in cobby and oranges